specs/CoreWebView2.CreateFromCOMObject.md

```diff
@@ -0,0 +1,70 @@
+CoreWebView2.CreateFromCOMObject
+===
+
+# Background
+The new [Unity WebView2](https://learn.microsoft.com/en-us/windows/mixed-reality/develop/advanced-concepts/webview2-unity-plugin)
+control creates and uses C++ COM to create and manage the ICoreWebView2* objects. However,
+Unity developers are often interacting with the Unity WebView2 control using C#/.NET. The Unity WebView2 control doesn't expose
+the CoreWebView2 directly to devs using the Unity WebView2 control, so when devs want to call an API on CoreWebView2, they
+have to rely on that API being exposed on the Unity WebView2 control, which then internally calls into CoreWebView2. This
+is in contrast to our other controls (like WPF WebView2 and Winforms WebView2 controls) which directly give access to their [CoreWebView2
+object](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.winforms.webview2.corewebview2?view=webview2-dotnet-1.0.2088.41), 
+allowing devs to call any API that exists or gets added to the CoreWebView2/ICoreWebView2_*. The Unity WebView2 control can't do this today,
+as they are unable to create a CoreWebView2 object that wraps an already existing COM object. To help implement this for Unity,
+we are adding a new static factory function on CoreWebView2 .NET class that will allow it to wrap an existing ICoreWebView2 COM object, instead
+of creating a new one underlying ICoreWebView2.
+
+# Examples
+## CoreWebView2.CreateFromCOMObject
+```c#
+public class MyWebView2Control
+{
+  ... // Regular control code
+
+  // A previously created ICoreWebView2
+  IntPtr _myCoreWebView2COMObject = ...;
```

Comment:
Do we have any code from Unity folks about how they're getting the COM object into their .NET code? It would be helpful to see this in determining that IntPtr is the correct type for the parameter.

Reply:
I've asked Jordan in the email thread. Will update with his response.

Reply:
Added the code from Unity into the example usage.